### ISSUE TITLE
Bad Linters and Checkers Error messages

### ISSUE THREAD

**Issue by adunuthulan** [treeverse/lakeFS#2380]
`make checks-validator` has uninformative error messages. 

```
Run make checks-validator
/opt/hostedtoolcache/go/1.16.2/x64/bin/go mod download
/opt/hostedtoolcache/go/1.16.2/x64/bin/go install github.com/deepmap/oapi-codegen/cmd/oapi-codegen
/opt/hostedtoolcache/go/1.16.2/x64/bin/go install github.com/golang/mock/mockgen
/opt/hostedtoolcache/go/1.16.2/x64/bin/go install github.com/golangci/golangci-lint/cmd/golangci-lint
/opt/hostedtoolcache/go/1.16.2/x64/bin/go install github.com/rakyll/statik
/opt/hostedtoolcache/go/1.16.2/x64/bin/go install google.golang.org/protobuf/cmd/protoc-gen-go
/home/runner/go/bin/golangci-lint run --out-format github-actions
checking gofmt...
Your code formatting is according to gofmt standards
/usr/bin/docker run --rm -v /home/runner/work/lakeFS/lakeFS:/mnt "treeverse/protoc:3.14.0" --proto_path=pkg/catalog --go_out=pkg/catalog --go_opt=paths=source_relative catalog.proto
Unable to find image 'treeverse/protoc:3.14.0' locally

[downloading image]

Digest: sha256:d8987e4c30be34ab833aa75ed76d84040025da95e7b2b30d60e8a149f7501c7c
Status: Downloaded newer image for treeverse/protoc:3.14.0
/usr/bin/docker run --rm -v /home/runner/work/lakeFS/lakeFS:/mnt "treeverse/protoc:3.14.0" --proto_path=pkg/graveler/committed --go_out=pkg/graveler/committed --go_opt=paths=source_relative committed.proto
/usr/bin/docker run --rm -v /home/runner/work/lakeFS/lakeFS:/mnt "treeverse/protoc:3.14.0" --proto_path=pkg/graveler --go_out=pkg/graveler --go_opt=paths=source_relative graveler.proto
git diff --quiet -- pkg/catalog/catalog.pb.go
git diff --quiet -- pkg/graveler/committed/committed.pb.go
git diff --quiet -- pkg/graveler/graveler.pb.go
git diff --quiet -- clients/python/lakefs_client/api
make: *** [Makefile:200: validate-client-python] Error 1
Error: Process completed with exit code 2.
```

I had not committed the regenerated python and java client files after updating the API, and was failing this test. The solution was pushing the updated client files, but this error message was not too helpful in determining that :)

**Comment by arielshaqed** [comment]
This seems to be for bad diffs (e.g., lint failures display beautifully).

**Comment by arielshaqed** [comment]
Fixed in #2686; thanks, @nopcoder !
